common/djangoapps/student/tests/tests.py

```diff
@@ -749,10 +750,10 @@ class EnrollInCourseTest(EnrollmentEventTestMixin, CacheIsolationTestCase):
     @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
     def test_enrollment(self):
         user = User.objects.create_user("joe", "joe@joe.com", "password")
-        course_id = SlashSeparatedCourseKey("edX", "Test101", "2013")
+        course_id = CourseKey.from_string("edX/Test101/2013")
         # Cannot be converted to CourseLocator or CourseKey.from_string because both do not support
```

Comment:
Oops, I need to remove this comment. When I first tried to convert it, I passed None for run and that threw an error. However, passing an empty string does work and is functionally equivalent.